openupgrade_scripts/scripts/hr_recruitment/16.0.1.1/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,65 @@
+---Models in module 'hr_recruitment'---
+new model applicant.send.mail [transient]
+
+# NOTHING TO DO: new transient model
+
+---Fields in module 'hr_recruitment'---
+hr_recruitment / hr.applicant             / interviewer_ids (many2many)   : NEW relation: res.users
+hr_recruitment / hr.applicant             / linkedin_profile (char)       : NEW
+hr_recruitment / hr.job                   / _order                        : _order is now 'sequence, name asc' ('sequence, state desc, name asc')
+hr_recruitment / hr.job                   / extended_interviewer_ids (many2many): NEW relation: res.users, isfunction: function, stored
+hr_recruitment / hr.job                   / interviewer_ids (many2many)   : NEW relation: res.users
+
+# NOTHING TO DO: new fields
```

Comment:
For readability, it's better to not leave the upper empty line, as this way, the comment stands in the middle of both the above and bottom blocks, not having clear to which one belongs:

```suggestion
# NOTHING TO DO: new fields
```

Reply:
:+1: